tools/data-migration-deployment.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Deploy Data Migration Using DM-Ansible
+summary: Use DM-Ansible to deploy the Data Migration cluster.
+category: tools
+---
+
+# Deploy Data Migration Using DM-Ansible
+
+DM-Ansible is a cluster deployment tool developed by PingCAP based on the [Playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#about-playbooks) feature of [Ansible](https://docs.ansible.com/ansible/latest/index.html) (an IT automation tool). This guide shows how to quickly deploy a Data Migration (DM) cluster using DM-Ansible.
+
+## Prepare
+
+Before you start, make sure you have the following machines as required.
+
+1. Several target machines that meet the following requirements:
+
+    - CentOS 7.3 or later, x86_64 architecture (AMD64)
+    - Network between machines
+    - Closing the firewall, or opening the service port
+
+2. A Control Machine that meets the following requirements:
```

Comment:
```suggestion
- A Control Machine that meets the following requirements:
```

Reply:
I also considered the list issue. Since there is a kind of sequence order, I'll keep the ordered list.